src/EventStore.ClusterNode/Program.cs

```diff
@@ -110,6 +110,10 @@ public class Program : ProgramBase<ClusterNodeOptions> {
 					"========================================================================================================\n");
 			}
 
+			if (opts.EnableAtomPubOverHTTP) {
+				Log.Warn("\n DEPRECATION WARNING: AtomPub over HTTP Interface has been deprecated in version 6. It is recommended to use GRPC instead.\n");
```

Comment:
Is it correct to call it version 6 here? Should this not be 20.02?

Reply:
@hayley-jean thanks changed to 20.02